synapse/cores/ram.py

```diff
@@ -14,9 +14,27 @@ def _initCortex(self):
         self.initSizeBy('le',self._sizeByLe)
         self.initRowsBy('le',self._rowsByLe)
 
+        self.initSizeBy('in',self._sizeByIn)
+        self.initRowsBy('in',self._rowsByIn)
+
         self.initSizeBy('range',self._sizeByRange)
         self.initRowsBy('range',self._rowsByRange)
 
+    def _sizeByIn(self, prop, valu, limit=None):
+        # HACK: for speed
+        data = dict(size=0)
+        def inc():
+            data['size'] += 1
+        [ inc() for r in self.rowsbyprop.get(prop,()) if r[2] in valu ]
```

Comment:
It does occur to me that {{r[2] in valu}} might benefit from converting from a list traversal to a hash lookup.